Classes/Userfuncs/CustomMenu.php

```diff
@@ -30,7 +30,7 @@ public function makeMenuArray(string $content, array $conf) : array
                             ],
                             [
                                 'title' => 'New York Office',
-                                '_OVERRIDE_HREF' => 'http://www.example.com',
+                                '_OVERRIDE_HREF' => 'https://www.example.com',
```

Comment:
No `.org` here?

Reply:
The menu entry is explicitly overwriting the default domain, therefore i chose to use the next domain in the list.

Reply:
OK, thanks.